projects/Flarum/services-example.nix

```
@@ -0,0 +1,4 @@
+{ ... }:
+{
+  programs.flarum.enable = true;
```

Since this is an example for the service

```suggestion
  services.flarum.enable = true;
```

---

Done.